### ISSUE TITLE
network output 0 and inf

### ISSUE THREAD

**Issue by txdat** [longcw/yolo2-pytorch#53]
i implemented your code with tensorflow (using pretrained vgg instead of darknet), after training with 100 epochs with VOC2007 (cls_loss use MSE, softmax), its output are all zeros or inf, what problems happened? thanks!

**Comment by txdat** [comment]
i think i missed 
    _boxes[:, :, 0:2] = 0.5
    _boxes[:, :, 2:4] = 1.0
    _box_mask = np.zeros([hw, num_anchors, 1], dtype=np.float) + 0.01

for training, is it important?

**Comment by txdat** [comment]
i forgot warmup epọchs, it works but cost lots of time to train pascal/voc
